go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/BurntSushi/toml v1.4.0
 	github.com/containers/libtrust v0.0.0-20230121012942-c1716e8a8d01
 	github.com/containers/ocicrypt v1.2.1
-	github.com/containers/storage v1.57.1
+	github.com/containers/storage v1.57.1-0.20250210200321-13a758ff4ecb
```

Comment:
Code LGTM, but this is a downgrade. Can we merge c/storage’s `1.57.1` into `main` ?

Reply:
yes we should https://github.com/containers/storage/pull/2255